plugins/times-of-day/frontend/public/javascripts/countly.views.js

```diff
@@ -264,436 +157,3 @@ app.addPageScript("/custom#", function() {
         });
     }
 });
-
-$(document).ready(function() {
```

Comment:
lets keep the following code, since this is required by custom dashboard.
removing this will break the times of day widget on custom dashboard.